types/meteor/globals/mongo.d.ts

```diff
@@ -186,29 +186,29 @@ declare module Mongo {
     interface CursorStatic {
         new <T>(): Cursor<T>;
     }
-    interface ObserveCallbacks {
-        added?(document: Object): void;
-        addedAt?(document: Object, atIndex: number, before: Object): void;
-        changed?(newDocument: Object, oldDocument: Object): void;
-        changedAt?(newDocument: Object, oldDocument: Object, indexAt: number): void;
-        removed?(oldDocument: Object): void;
-        removedAt?(oldDocument: Object, atIndex: number): void;
-        movedTo?(document: Object, fromIndex: number, toIndex: number, before: Object): void;
-    }
-    interface ObserveChangesCallbacks {
-        added?(id: string, fields: Object): void;
-        addedBefore?(id: string, fields: Object, before: Object): void;
-        changed?(id: string, fields: Object): void;
-        movedBefore?(id: string, before: Object): void;
+    interface ObserveCallbacks<T> {
```

Comment:
I believe this is a breaking change for existing projects. Let's avoid that: please provide a default value for `T` (same also for the second interface below)